docs/cookbook/audionotification/index.rst

```diff
@@ -0,0 +1,61 @@
+Audio Notification
+==================
+
+Problem
+-------
+
+Your plugin needs to provide some form of audio notification or audible alert.
+
+
+Solution
+--------
+
+Use the OpenSphere AudioManager. This is a singleton, and the `play()` function takes a label for the sound file to play:
+
+.. literalinclude:: src/audioalertplugin.js
+  :caption: AudioManager usage
+  :linenos:
+  :lines: 30-31
+  :language: javascript
+
+There are three ways to provide the sound files - config settings, programmatically, or by user upload.
+
+A config setting entry is shown below:
+
+.. literalinclude:: src/settings.json
+  :caption: AudioManager sound file configuration
+  :linenos:
+  :language: json
+
+A programmatic approach is shown below:
+
+.. code-block:: javascript
+
+  var audioManager = os.audio.AudioManager.getInstance();
+  audioManager.addSound(os.ROOT + 'sounds/cowbell.wav', 'label');
+  audioManager.play('label');
+
+User upload uses the normal Import Data dialog. Note that it only works if the target is a HTTP or HTTPS URL.
+
```

Comment:
This is true only if the application is served via web. If you are using something like Electron.io to package an app for desktop installation, `file://` should also work.

Reply:
I thought I saw some code that checked for this. Will double check.